src/DBus.vala

```diff
@@ -110,6 +110,17 @@ public class Notifications.Server : Object {
         } else {
             var notification = new Notifications.Notification (app_name, app_icon, summary, body, actions, hints);
 
+            // Silence "Automatic suspend. Suspending soon because of inactivity." notifications
+            // These values and hints are taken from gnome-settings-daemon source code
```

Comment:
Can we add a link to https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/blob/master/plugins/power/gsd-power-manager.c#L356 for future convenience :)